docs/docs/widgets/password-input.md

```diff
@@ -0,0 +1,34 @@
+# Password Input
+
+A Password Input widget provides a way for the users to securely enter a password. The Password Input is a one-line plain text editor in which the text is obscured so that it cannot be read, by replacing each character with an asterisk ("*") symbol.
+
+<img class="screenshot-full" src="/img/widgets/password-input/password-input.gif" alt="ToolJet - Widget Reference - Password Input" height="420"/>
+
+#### Properties
+
+| properties      | description |
+| ----------- | ----------- |
+| Placeholder |  It specifies a hint that describes the expected value.|
+
+#### Validation
+
+| Validation      | description |
+| ----------- | ----------- |
+| Regex | Use this field to enter a Regular Expression that will validate the password constraints. |
+| Min length | Enter the number for a minimum length of password allowed.|
+| Max length | Enter the number for the maximum length of password allowed. |
+| Custom validation | If the condition results true, the validation passes, and if false, it will display the string. For example: `{{components.passwordInput1.value === 'something' ? true: 'value should be something'}}` |
```

Comment:
`If the condition results true, the validation passes, and if false, it will display the string` -> `If the condition is true, the validation passes, otherwise return a string that should be displayed as the error message`